docs/Translating.md

```diff
@@ -4,10 +4,8 @@ As with any other project, we expect you to be familiar with particular Transmis
 
 General rule is that new localizations are only included in next release if they are at least 95% complete. Transifex (see below) offers a review functionality, so if you feel like it and are not afraid of responsibility — don't hesitate to ask and we could then have a different deal based on reviewed percentage instead (nothing is set in stone as of now).
 
-## Qt and GTK+ clients
-Translations for Qt and GTK+ clients are managed on Transifex. To help localize, or add your new localization, you'll need to [register on Transifex and join the translation team](https://docs.transifex.com/getting-started/translators) for [Transmission project](https://www.transifex.com/transmissionbt/transmissionbt).
+Translations for Mac, Qt and GTK+ clients are managed on Transifex. To help localize, or add your new localization, you'll need to [register on Transifex and join the translation team](https://docs.transifex.com/getting-started/translators) for [Transmission project](https://www.transifex.com/transmissionbt/transmissionbt).
 
 _GTK+ client translations were previously managed on [Launchpad](https://translations.launchpad.net/transmission/trunk/+pots/transmission), but this is no longer the case. No phrases will be updated there, and none of your translations made there will be integrated back into Transmission._
 
-## Mac client
-Parts of Mac client translations have been recently uploaded to Transifex as well. We are still in the process of making all of the translations accessible there, including plural forms support and phrases currently only present in .xib files. It will all be there prior to next major release, so stay tuned. You could start localizing now (follow the same steps as above), but be prepared for some inconveniences along the way.
+_Mac client translations are almost entirely accessible on Transifex, but we are still in the process of including plural forms support and phrases currently only present in .xib files._
```

Comment:
Is part about .xib files still relevant? I thought this was solved with #3405.

Reply:
OK. I've now only kept the plural form part.

Reply:
Maybe mention that they're for the Mac client?.. Also, #2250 is a specific issue while proper plural support for Mac client in particular (via .stringsdict, IIUC) will need more work that's probably better registered separately.

Reply:
What about I just delete the sentence?